README.md

```diff
@@ -114,6 +114,30 @@ flags that queries all data rows with that flag.
 This method will create the Hive schema, which is the collection of tables
 flags, flagged_data, and service_periods.
 
+#### `int lookup_flag_id(flag_name)`
+
+This method expects a string `flag_name` and will return the numerical id
+of the corresponding flag, if it exists. None will be returned if the dictionary
+has not been initialized.
+
+#### `client_instance._init_flag_dict()`
+
+Query the database for all present flag-types. This information is made available
+for translating queries utilizing flag names into queries with flag ids. This
+functionality is performed on the program-side instead of the database-side
+in order to allow the end-user to view what flags exist and write command-
+line queries with sensible names. NOTE: Current behavior only runs this
+operation and initializes the dictionary as-needed. This means in use cases
+where translating flag names to ids is not needed, this table does not get
+initialized.
```

Comment:
Could you drop this documentation from the README and put it inline? My idea with the client docs in the README is that the client is what the users will interface with so the public members should be very well documented, but we don't need to included hidden members.